.eslintrc.json

```diff
@@ -12,14 +12,16 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],
+  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb", "plugin:prettier/recommended"],
   "settings": {
     "react": {
       "version": "detect"
     }
   },
-  "plugins": ["react", "jsx-a11y", "import"],
+  "plugins": ["react", "jsx-a11y", "prettier"],
   "rules": {
+    "arrow-body-style": "off",
+    "prefer-arrow-callback": "off", 
```

Comment:
were these two rules added by accident?

Reply:
No, those were deliberate as I've found that the eslint plugin for prettier extension has known issues associated with those specific es core rules :/ https://github.com/prettier/eslint-plugin-prettier#arrow-body-style-and-prefer-arrow-callback-issue

Reply:
thanks for the explanation, didn't know that